src/App.js

```diff
@@ -1,12 +1,171 @@
+                                onChange={(e) => setZip(e.target.value)}
+                            />
+                        </label>
+                        <label htmlFor="bezorgfreq">Bezorgfrequentie
+                            <select name="bezorgfreq"
+                                    id="bezorgfreq"
+                                    value={delivFreq}
+                                    onChange={(e) => setDelivFreq(e.target.value)}
+                            >
+                                <option value="iedere week">Iedere week</option>
+                                <option value="om de week">Om de week</option>
+                                <option value="iedere maand">iedere maand</option>
+                            </select>
+                        </label>
+                        <div className="delivtimecontainer">
+                            <label htmlFor="delivTime1">
+                                <Info
+                                    id="delivTime1"
+                                    type="radio"
+                                    defaultChecked={true}
+                                    name="delivTime"
+                                    value="Overdag"
+                                    onClick={(e) => setDelivTime(e.target.value)}
+                                />Overdag
+                            </label>
+                            <label htmlFor="delivTime2">
+                                <Info
+                                    id="delivTime2"
+                                    type="radio"
+                                    name="delivTime"
+                                    value="'s Avonds"
+                                    onClick={(e) => setDelivTime(e.target.value)}
+                                />'s Avonds
+                            </label>
+                        </div>
+                        <label htmlFor="comment">Opmerkingen</label>
+                        <textarea
+                            className="comment"
+                            name="comment"
+                            id="comment"
+                            cols="30" rows="5"
+                            value={comment}
+                            onChange={(e) => setComment(e.target.value)}
+                        >
+                        </textarea>
+                        <label htmlFor="terms">
+                            <Info
+                                id="terms"
+                                type="checkbox"
+                                name="terms"
+                                value={terms}
+                                onClick={(e) => setTerms(!terms)}
+                            />Ik ga akkoord met de voorwaarden
+                        </label>
+                    </form>
+                    <div className="sendContainer">
+                        {total.valueOf() > 0 && <p className="greet">Eet smakelijk!</p>}
```

Comment:
Je had wellicht deze tekst ook in een alert() kunnen zetten en pas aanroepen in de handleClick van de button

Reply:
Ik heb een derde versie gemaakt waar het wel gelukt is uiteindelijk. Er waren nog een paar extra gedachtenstappen voor nodig.